### PR TITLE
fix(doc): more accurate translation

### PR DESCRIPTION
**Pull request by xianghongai**
Some of the previous translations are incorrect.

### CODE REVIEW

File: docs/zh/api/README.md [hunk 1]
```diff
@@ -40,12 +40,12 @@ const store = new Vuex.Store({ ...options })
 
   ``` js
   {
-    state,      // 等同于 `store.state`，若在模块中则为局部状态
-    rootState,  // 等同于 `store.state`，只存在于模块中
```
Comment (Alanscut) [inline docs/zh/api/README.md:44]: The current translation developer could probably understand

**Review by Alanscut — COMMENTED** [review]
@Justineo I think Mr. Gu could give a more authoritative opinion.

**Comment by kiaking** [comment]
@phanan Could you review this one please 🙏 🙏 🙏

**Comment by phanan** [comment]
Sorry @kiaking, I don't think my rusty Chinese is good enough for this 😅.

**Comment by kiaking** [comment]
Oh 😅   @Justineo would it be possible to take a quick look on this 🙏

**Comment by Alanscut** [comment]
@Justineo has reviewed my PR #1965 in detail and make more than 60 comments, but didn't ask for changes to above translations. see in [zh/api/index.md](https://github.com/vuejs/vuex/pull/1965/files?file-filters%5B%5D=.md#diff-cf95a910ff1ed91ca6fe0133cbdebfff010baee886adac3df9c9db93fe380c5e)

so maybe we should keep the way it was 🤔

**Comment by Justineo** [comment]
I believe the current version is good enough. Let's keep it as-is.

**Comment by kiaking** [comment]
OK. @xianghongai Thank you so much for the PR! As of now, we'll drop this PR and leave the doc as is. But really appreciate your effort! 🤝

**Pull request closed without merging**